### DISCUSSION TITLE
Forbidden

### DISCUSSION THREAD

**Discussion by yazigegeda** [coder/code-server#5797] [General]
![Screenshot_2022-11-20-12-46-04-85_84d3000e3f4017145260f7618db1d683](https://user-images.githubusercontent.com/63853093/202886265-3e8c2d7b-16ea-467f-ad72-e275ad0fdaef.jpg)

I visited 127.0.0.1:8080 and successfully entered the password, but Forbidden appeared

**Comment by lg-nineoneninethreenine** [comment]
It must be fixed now, try a PKG update.
